Framework/include/QualityControl/ObjectsManager.h

```diff
@@ -20,6 +20,7 @@
 #include "QualityControl/MonitorObject.h"
 #include "QualityControl/Quality.h"
 #include "QualityControl/TaskConfig.h"
+#include "ServiceDiscovery.h"
```

Comment:
Why not "QualityControl/ServiceDiscovery.h", since it is inside QualityControl?

Reply:
it was a mistake. Originally the header was not distributed.